assets/src/edit-story/components/panels/textStyle.js

```diff
@@ -0,0 +1,287 @@
+                fontWeights: currentFontWeights,
+                fontFallback: currentFontFallback,
+              });
+            }}
+          />
+        )}
+      </Row>
+      <Row>
+        <SelectMenu
+          ariaLabel={__('Font style', 'web-stories')}
+          options={fontStyles}
+          isMultiple={fontStyle === ''}
+          value={state.fontStyle}
+          onChange={(value) => setState({ ...state, fontStyle: value })}
+        />
+        <Space />
+        <BoxedNumeric
+          ariaLabel={__('Font size', 'web-stories')}
+          value={state.fontSize}
+          isMultiple={fontSize === ''}
+          min={MIN_FONT_SIZE}
+          max={MAX_FONT_SIZE}
+          flexBasis={58}
+          textCenter
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+      </Row>
+      <Row>
+        {/* TODO: Add vertical offset logic */}
+        <ExpandedNumeric
+          ariaLabel={__('Vertical offset', 'web-stories')}
+          value={state.fontSize}
+          suffix={<VerticalOffset />}
+          isMultiple={fontSize === ''}
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+        <Space />
+        {/* TODO: Add horizontal offset logic */}
+        <ExpandedNumeric
+          ariaLabel={__('Horizontal offset', 'web-stories')}
+          value={state.fontSize}
+          suffix={<HorizontalOffset />}
+          isMultiple={fontSize === ''}
+          onChange={(value) =>
+            setState({ ...state, fontSize: parseInt(value) })
+          }
+        />
+      </Row>
+      <Row>
+        <SelectMenu
+          label={__('Alignment', 'web-stories')}
+          options={alignmentOptions}
+          isMultiple={'' === textAlign}
+          value={state.textAlign}
+          onChange={(value) => setState({ ...state, textAlign: value })}
+        />
+      </Row>
+      <Row>
+        <Label>{__('Text', 'web-stories')}</Label>
+        <Color
+          value={state.backgroundColor}
+          onChange={(value) => setState({ ...state, backgroundColor: value })}
+          opacity={1}
+        />
+      </Row>
+      <Row>
+        <Label>{__('Padding', 'web-stories')}</Label>
+        <BoxedNumeric
+          suffix={_x('H', 'The Horizonatal padding', 'web-stories')}
+          value={state.padding}
+          onChange={() => {}}
+        />
+        <Space />
+        <Toggle
+          icon={<Locked />}
+          uncheckedIcon={<Unlocked />}
+          value={lockRatio}
+          isMultiple={false}
+          onChange={(value) => {
+            setLockRatio(value);
+          }}
+        />
+        <Space />
+        <BoxedNumeric
+          suffix={_x('V', 'The Vertical padding', 'web-stories')}
+          value={state.padding}
+          onChange={() => {}}
+        />
+      </Row>
+      {/* TODO: Update padding logic */}
+      {/* <InputGroup
```

Comment:
Remove this commented out code? Or what's its purpose here?

Reply:
There was original logic for padding adjustment. Removed and moved TODO to upper level.